src/main/steps/c100-rebuild/check-your-answers/PayAndSubmitPostController.ts

```diff
@@ -1,45 +0,0 @@
-import autobind from 'autobind-decorator';
-import { Response } from 'express';
-
-import { PaymentErrorContext } from '../../../app/case/definition';
-import { AppRequest } from '../../../app/controller/AppRequest';
-import { AnyObject, PostController } from '../../../app/controller/PostController';
-import { Form, FormFields, FormFieldsFn } from '../../../app/form/Form';
-import { PaymentHandler } from '../../../modules/payments/paymentController';
-import { C100_CHECK_YOUR_ANSWER } from '../../../steps/urls';
-
-@autobind
-export default class PayAndSubmitPostController extends PostController<AnyObject> {
-  constructor(protected readonly fields: FormFields | FormFieldsFn) {
-    super(fields);
-  }
-
-  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
-    try {
-      req.session.paymentError = { hasError: false, errorContext: null };
-      if (req.body.saveAndComeLater) {
-        this.saveAndComeLater(req, res, req.session.userCase);
-      } else {
-        const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase, req) : this.fields;
-        const form = new Form(fields);
-        const { ...formData } = form.getParsedBody(req.body);
-        req.session.errors = form.getErrors(formData);
-        if (req.session.errors && req.session.errors.length) {
-          return super.redirect(req, res, C100_CHECK_YOUR_ANSWER);
-        }
-
-        /** Invoke create payment
-         * 1. Create only service request for case with help with fees opted
-         * 2. Create service request & payment request ref in case of pay & submit
```

Comment:
what happened?

Reply:
This controller is now distributed 
one for pcq and other for handling the process after pcq questionnaire